metaflow/plugins/resources_decorator.py

```diff
@@ -33,4 +33,10 @@ class ResourcesDecorator(StepDecorator):
     """
 
     name = "resources"
-    defaults = {"cpu": "1", "gpu": "0", "memory": "4096", "shared_memory": None}
+    defaults = {
+        "cpu": "1",
+        "gpu": "0",
+        "memory": "4096",
+        "shared_memory": None,
```

Comment:
#1504 opened for this, currently not yet supported for Kubernetes.

Reply:
https://outerbounds-community.slack.com/archives/C020U025QJK/p1691751137791999 earlier discussions with context on why `disk` has been missing.